src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -674,6 +674,45 @@ public function unitAction(Request $request, $action, $id)
         }
     }
 
+    public function exportAction($_format)
+    {
+        // init vars
+        $productProvider = $this->container->get('prestashop.core.admin.data_provider.product_interface');
+        /* @var $productProvider ProductInterfaceProvider */
```

Comment:
The annotation works in Eclipse and PHPStorm in this way. If there is documentation on the official syntax, I will follow it.